commands/run.sh

```diff
@@ -40,8 +40,16 @@ prebuilt_service_overrides=()
 prebuilt_services=()
 
 # We look for a prebuilt images for all the pull services and the run_service.
+prebuilt_image_override="$(plugin_read_config RUN_IMAGE)"
 for service_name in "${prebuilt_candidates[@]}" ; do
-  if prebuilt_image=$(get_prebuilt_image "$service_name") ; then
+  prebuilt_image=$(get_prebuilt_image "$service_name") || :
+
+  # override run_service prebuilt_image if set
+  if [[ -n "$prebuilt_image_override" ]] && [[ "$service_name" == "$run_service" ]] ; then
+    prebuilt_image="$prebuilt_image_override"
+  fi
+
+  if [[ -n "$prebuilt_image" ]] ; then
     echo "~~~ :docker: Found a pre-built image for $service_name"
```

Comment:
An alternative to this would be:

```suggestion
prebuilt_image_override="$(plugin_read_config RUN_IMAGE)"
for service_name in "${prebuilt_candidates[@]}" ; do
  if [[ "$service_name" == "$run_service" ]] && [[ -n "$prebuild_image_override" ]]; then
    echo "~~~ :docker: Overriding run image for $service_name"
    prebuilt_image="$prebuild_image_override"
  elif prebuilt_image=$(get_prebuilt_image "$service_name") ; then
     echo "~~~ :docker: Found a pre-built image for $service_name"
  fi

  if [[ -n "$prebuilt_image" ]] ; then
```